### PR TITLE
fixes #8483: copy WB data from Pentax K20D to Samsung GX20 + copy adobe coeff from…

### LINKED ISSUE #8483: Samsung GX10/GX20 support (Pentax K10D/K20D clones)
Hello,

I have old photos taken with a Samsung GX20, and there is no WB nor noise profile for this camera in darktable.
Unfortunately, I do not own more this camera to do the needed pictures.
However, Pentax's K10D and K20D profiles exists, and Samsung's GX10 and GX20 are clones of these Pentax cameras.
I tried to copy-paste the noise profile of Pentax K20D to a Samsung GX20 profile, and it seems working not badly.

Do you think it is acceptable to copy all the data concerning Pentax's K10D and K20D to Samsung's GX10 and GX20, allowing better support of these cameras? (that is, WB presets, noise profile and custom matrix)

Some info:
- Samsung's raws are DNG format.
- These bodies are issued from a Pentax/Samsung partnership, if I am not wrong the sensor is from Samsung and body design from Pentax.

Regards

### PR DESCRIPTION
**Pull request by jiemdev**
fixes #8483:
copy WB data from Pentax K20D to Samsung GX20 + copy adobe coeff from Pentax K10D to Samsung GX10 + copy custom color matrix from Pentax K20D to Samsung GX20 + copy noise profiles from K10D and K20D to GX10 and GX20

### CODE REVIEW

**Comment by johnny-bit** [comment]
Can you add "fixes #8483" to your commit & 1st message in PR?

Otherwise it should be OK.

File: src/common/colormatrices.c [hunk 1]
```diff
@@ -181,6 +181,9 @@ static dt_profiled_colormatrix_t dt_profiled_colormatrices[] =
   // Wolfgang Kuehnel, Wolf Faust IT8, strobe, well lit
   { "Minolta Dynax 5D",             { 910599, 389618,  20218}, { 330353, 1223724, -116943}, { 24384, -307190, 1156891}, {604309, 629196, 525848}},
 
+  // copied from Pentax K20d
+  { "Samsung GX20",                  {1008652, 388794, -36346}, { 162323, 1113815, -341446}, { 81863, -214325, 1431107}, {664963, 685287, 527252}},
```
Comment (LebedevRI) [inline src/common/colormatrices.c:185]: Please remove this, the camera only produces DNG's.

File: src/external/adobe_coeff.c [hunk 2]
```diff
@@ -622,6 +622,7 @@ static void dt_dcraw_adobe_coeff(const char *name, float cam_xyz[1][12])
     { "Samsung NX1", { 10686,-4042,-1052,-3595,13238,276,-464,1259,5931 } },
     { "Samsung WB2000", { 12093,-3557,-1155,-1000,9534,1733,-22,1787,4576 } },
     { "Samsung GX-1", { 10504,-2438,-1189,-8603,16207,2531,-1022,863,12242 } },
+    { "Samsung GX10", { 9566,-2863,-803,-7170,15172,2112,-818,803,9705 } }, /* copied from Pentax K10D */
```
Comment (LebedevRI) [inline src/external/adobe_coeff.c:625]: Please remove this, the camera only produces DNG's.

Comment (jiemdev) [reply src/external/adobe_coeff.c:625]: In this case, I guess I should remove the line concerning the Samsung GX20?
It does only produces DNG as raw format too, but a line already exists in this file for it.

Comment (LebedevRI) [reply src/external/adobe_coeff.c:625]: Hmm. I think so, yes.

Comment (jiemdev) [reply src/external/adobe_coeff.c]: ok, done!

**Review by johnny-bit — APPROVED** [review]
IMO looks good

**Comment by TurboGit** [comment]
Thanks !

**Pull request merged**